HousingRepairsOnlineApi/Properties/launchSettings.json

```diff
@@ -42,7 +42,8 @@
+        "SOR_CONFIGURATION": "[{\"value\":\"kitchen\",\"display\":\"Kitchen\",\"options\":[{\"value\":\"cupboards\",\"display\":\"Cupboards, including damaged...\",\"options\":[{\"value\":\"doorHangingOff\",\"display\":\"Hanging Door\",\"sorCode\":\"<enter sor code>\"},{\"value\":\"doorMissing\",\"display\":\"Missing door\",\"sorCode\":\"<enter sor code>\"}]},{\"value\":\"worktop\",\"display\":\"Damaged worktop\",\"sorCode\":\"<enter sor code>\"},{\"value\":\"dampOrMould\",\"display\":\"Damp or mould\",\"options\":[{\"value\":\"emergency\",\"display\":\"Damp or mould caused by a leak\"},{\"value\":\"dampOrMould\",\"display\":\"Damp or mould caused by something else\",\"sorCode\":\"<enter sor code>\"}]},{\"value\":\"unableToBook\",\"display\":\"Heating or hot water\"},{\"value\":\"electrical\",\"display\":\"Electrical, including extractor fans and lightbulbs\",\"options\":[{\"value\":\"lights\",\"display\":\"Lights\",\"sorCode\":\"<enter sor code>\"},{\"value\":\"sockets\",\"display\":\"Sockets\",\"sorCode\":\"<enter sor code>\"}]}]},{\"value\":\"bathroom\",\"display\":\"Bathroom\",\"options\":[{\"value\":\"bath\",\"display\":\"Bath, including taps\",\"options\":[{\"value\":\"bathTaps\",\"display\":\"Bath taps\",\"sorCode\":\"<enter sor code>\"},{\"value\":\"sealAroundBath\",\"display\":\"Seal around bath\",\"sorCode\":\"<enter sor code>\"}]}]},{\"value\":\"bedroom\",\"display\":\"Bedroom\",\"options\":[{\"value\":\"damagedOrStuckDoors\",\"display\":\"Damaged or stuck doors\",\"options\":[{\"value\":\"internalDoorIssue\",\"display\":\"Internal door issue, including hinges, handle, sticking\",\"sorCode\":\"<enter sor code>\"},{\"value\":\"notEligibleNonEmergency\",\"display\":\"Adjusting a door after a carpet fitting\"},{\"value\":\"emergency\",\"display\":\"Door is on fire\"},{\"value\":\"unableToBook\",\"display\":\"Don't like door colour\"}]}]}]",
+        "ALLOWED_APPOINTMENT_SLOTS": "[ { \"startTime\": \"08:00:00\", \"endTime\": \"12:00:00\" }, { \"startTime\": \"12:00:00\", \"endTime\": \"18:00:00\" } ]"
```

Comment:
We didn't need to add school hours to this did we? Or would this be added in later config?

Reply:
This is an example value for the configuration and the Newark value for `ALLOWED_APPOINTMENT_SLOTS` will be provided via configuration.